apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/internal/AndroidTaskConfigurator.kt

```diff
@@ -43,6 +41,7 @@ object AndroidTaskConfigurator {
     return container
   }
 
+  // TODO: make this lazy
```

Comment:
I would put a link to this issue. https://github.com/apollographql/apollo-android/issues/1454 Since it is blocked, it would be useful.

Reply:
👍 will do